doc/iterations/2025.2/add_loading_indicators_for_long_actions.adoc

```diff
@@ -0,0 +1,56 @@
+= Add loading indicators for long actions
+
+== Problem
+
+Some long actions do not have a loading indicator.
+
+The lack of feedback for the user is an ui/ux issue.
+
+== Key Result
+
+All long actions should provide some kind of loading indicator.
+
+With MUI we can provide 3 types of indicator depending on the situation :
+
+1 - A circular progress bar used to replace a button after a click
+
+2 - A linear progress bar used to replace a component which contains several other component
+
+3 - A skeleton progress that represent a placeholder shape during loading
+
+=== Scenario
+
+==== An user want to open a representation
+
+- Since representations can have many different shapes, we'll show a linear progress bar by default.
+- On table representation, form representation and tree representation we could use a skeleton shape.
+
+==== An user want to execute a long action on a representation
+
+- Afer 1sec, we'll replace the current representation with the same loading indicator as the one used during opening.
+
+==== An user want to execute a long action on a diagram representation
+
+- We should not replace the composant since it would negatively impact performances, but we will still show a linear progress bar on top.
+- We can also temporarily put the diagram in read only to make sure that the user can't perform another action before the first one is finished.
+
+==== An user is impacted by a long action triggered by another user
+
+- We use the same strategy as if he was the one to execute the action.
+
+
+=== Breadboarding
+
+- Progress & Skeleton components of MUI will be used.
+
+
+=== Cutting backs
+
+- Using suspens API.
```

Comment:
- suspense: https://react.dev/reference/react/Suspense

Reply:
I'll even move those two items to no-gos because there is not a single chance that we will move forward on those items now